extensions/http/src/main/java/jolie/net/HttpProtocol.java

```diff
@@ -1388,7 +1381,8 @@ private String cutBeforeQuerystring( String requestPath ) {
 		return requestPath.split( "\\?", 2 )[ 0 ];
 	}
 
-	private void recv_checkReceivingOperation( HttpMessage message, DecodedMessage decodedMessage ) {
+	private Optional< RequestErrorCommMessage > recv_checkReceivingOperation( HttpMessage message,
```

Comment:
Are we start to use return error as a value practice? I guess waiting for review from @fmontesi